### PR TITLE
Dont panic if input is bad

### LINKED ISSUE #6898: API breaking change in network version numbering
### Checklist

- [X] This is **not** a security-related bug/issue. If it is, please follow please follow the [security policy](https://github.com/filecoin-project/lotus/security/policy).
- [X] This is **not** a question or a support request. If you have any lotus related questions, please ask in the [lotus forum](https://github.com/filecoin-project/lotus/discussions).
- [X] This is **not** a new feature request. If it is, please file a [feature request](https://github.com/filecoin-project/lotus/issues/new?assignees=&labels=need%2Ftriage%2Ckind%2Ffeature&template=feature_request.yml) instead.
- [X] This is **not** an enhancement request. If it is, please file a [improvement suggestion](https://github.com/filecoin-project/lotus/issues/new?assignees=&labels=need%2Ftriage%2Ckind%2Fenhancement&template=enhancement.yml) instead.
- [X] I **have** searched on the [issue tracker](https://github.com/filecoin-project/lotus/issues) and the [lotus forum](https://github.com/filecoin-project/lotus/discussions), and there is no existing related issue or discussion.
- [X] I am running the [`Latest release`](https://github.com/filecoin-project/lotus/releases), or the most recent RC(release canadiate) for the upcoming release or the dev branch(master), or have an issue updating to any of these.
- [X] I did not make any code changes to lotus.

### Lotus component

lotus JSON-RPC API

### Lotus Version

```text
master
```

### Describe the Bug

See: https://github.com/filecoin-project/go-state-types/pull/30#issuecomment-887934920

Changing the network version numbers (1 -> 100, etc.) is technically API breaking. We either need to bump the API version, or handle this somehow.

Now, we _can_ handle one half pretty easily: if we're a "lite" node, we can "upgrade" version numbers by multiplying all version numbers less than 100 by 100.

The tricky part is that anything consuming the API will still be broken. Unfortunately, there likely is no good way to fix that without bumping the API version.

### Logging Information

```text
None.
```

### Repo Steps

_No response_

### PR DESCRIPTION
**Pull request by whyrusleeping**
Will mostly fix #6898 when it compiles. (at least, will prevent things from panicking and give the user more of an idea of what is going wrong).

### CODE REVIEW

**Comment by ZenGround0** [comment]
Need to fix actors policy gen.

File: cmd/lotus-sim/simulation/blockbuilder/blockbuilder.go [hunk 1]
```diff
@@ -271,7 +271,7 @@ func (bb *BlockBuilder) StateManager() *stmgr.StateManager {
 }
 
 // ActorsVersion returns the actors version for the target block.
-func (bb *BlockBuilder) ActorsVersion() actors.Version {
+func (bb *BlockBuilder) ActorsVersion() (actors.Version, error) {
 	return actors.VersionForNetwork(bb.NetworkVersion())
```
Comment (Stebalien) [inline cmd/lotus-sim/simulation/blockbuilder/blockbuilder.go:275]: I'd panic in the error case here. This is a testing tool.

**Review by Stebalien — APPROVED** [review]
One nit you can ignore.

File: cmd/lotus-sim/simulation/blockbuilder/blockbuilder.go (hunk 1 above)
Comment (Stebalien) [reply cmd/lotus-sim/simulation/blockbuilder/blockbuilder.go:275]: Eh, not worth blocking on.

**Pull request merged**